.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,17 @@
+name: Trivy
+
+on:
+  push:
+    branches:
+      - master
+      - branch/*
```

Comment:
Shouldn't this be `branch/**`?

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-filters

Reply:
I cargo-culted this from other workflows in this repo. I presume it's meant to match the `branch/vNN` branches, but not anything with a slash in it (e.g. `branch/v10/something_else`); I don't see any of these, but it also seems like something we don't want to match.